api/marketdata.go

```diff
@@ -37,12 +36,11 @@ type Coin struct {
 }
 
 func SetupMarketAPI(router gin.IRouter, db storage.Market, charts *market.Charts, ac assets.AssetClient) {
-	router.Use(ginutils.TokenAuthMiddleware(viper.GetString("market.auth")))
 	// Ticker
 	router.POST("/ticker", getTickersHandler(db))
 	// Charts
-	router.GET("/charts", gincache.CacheMiddleware(time.Minute*10, getChartsHandler(charts)))
-	router.GET("/info", gincache.CacheMiddleware(time.Minute*5, getCoinInfoHandler(charts, ac)))
+	router.GET("/charts", middleware.CacheMiddleware(minute*5, getChartsHandler(charts)))
```

Comment:
just curious: how often is the `time_start` parameter here the same roughly aka what do you roughly expect the cache hit rate to be?

Reply:
Yep, approximately